src/js/constants.js

```diff
@@ -29,6 +29,13 @@ var exports = {
   USER_BLOCK: "user_block",
   USER_COOKIE_BLOCK: "user_cookieblock",
 
+  BLOCKED_ACTIONS: [
```

Comment:
Oops, sorry, my previous suggestion is wrong, "USER_BLOCK" is just a string then. You can do either
```javascript
let BLOCKED = {};
BLOCKED[exports.COOKIEBLOCK] = true;
BLOCKED[exports.BLOCK] = true;
...
exports.BLOCKED = BLOCKED;
```
or (using ES6 computed property names),
```javascript
exports.BLOCKED = {
  [exports.COOKIEBLOCK]: true,
  [exports.BLOCK]: true,
  ...
};
```